build/index.native.js

```diff
@@ -0,0 +1,6 @@
+const Recoil = require('../src/Recoil_index');
```

Comment:
Needs copyright header

Reply:
It's not clear from the GitHub PR where this file lives in `www` and if it may also be a candidate for the reaper?

Reply:
don't worry -- this will be in scripts like other build scripts